CMakeLists.txt

```diff
@@ -611,6 +612,11 @@ if (ENABLE_KUBERNETES)
   createExtension(KUBERNETES-EXTENSIONS "KUBERNETES EXTENSIONS" "This enables Kubernetes support" "extensions/kubernetes")
 endif()
 
+## Add the test-processors extension
+if (ENABLE_ALL OR NOT SKIP_TESTS OR ENABLE_TEST_PROCESSORS)
+	createExtension(TEST-PROCESSORS-EXTENSION "TEST-PROCESSORS EXTENSION" "This enables processors purely for testing purposes" "extensions/test-processors")
+endif()
```

Comment:
So you just want the condition to be `if (NOT SKIP_TESTS)` ?
`ENABLE_TEST_PROCESSORS` was introduced to avoid building all unit tests in docker integration tests, where `SKIP_TESTS` is true by default. But these processors are still needed for integration tests.

Reply:
Ok, this makes sense, in this case could you please add a comment about this relationship?

And consider removing ENABLE_ALL, since these are not end-user feature processors.

Reply:
Done.